uiowa/uiowa
# Feature: Fix missing padding adjustment for left aligned banners

# How to test

Check to make sure the left-aligned padding is removed as seen in the screenshots below. 

The update for https://github.com/uiowa/uiowa/pull/2950/files misplaced a closing tag for removing left padding for left aligned banners.  This pr fixes that closing tag.    It also fixes some padding for left-aligned banners within the edge-to-edge container.  

Before:

<img width="1469" alt="Screen Shot 2021-02-16 at 12 59 12 PM" src="https://user-images.githubusercontent.com/1036433/108108900-dc45ea80-7056-11eb-8f44-c9c02022b793.png">

After:

<img width="1473" alt="Screen Shot 2021-02-16 at 12 59 29 PM" src="https://user-images.githubusercontent.com/1036433/108108920-e2d46200-7056-11eb-8132-16b0e56d5fa2.png">


Also fixes button alignment for AoS:

Before:

<img width="574" alt="Screen Shot 2021-02-16 at 12 58 46 PM" src="https://user-images.githubusercontent.com/1036433/108108973-f253ab00-7056-11eb-927d-9f75a87873b5.png">

After:

<img width="549" alt="Screen Shot 2021-02-16 at 12 58 57 PM" src="https://user-images.githubusercontent.com/1036433/108108976-f2ec4180-7056-11eb-9d23-00d25c770271.png">
